src/sql/workbench/browser/modelComponents/treeViewDataProvider.ts

```diff
@@ -32,4 +49,29 @@ export class TreeViewDataProvider extends vsTreeView.TreeViewDataProvider implem
 
 	refresh(itemsToRefreshByHandle: { [treeItemHandle: string]: ITreeComponentItem }) {
 	}
+
+	getChildren(treeItem?: ITreeComponentItem): Promise<ITreeComponentItem[]> {
```

Comment:
So I don't think you need to override this one.

Reply:
Jeez, yeah don't know why I did it like this. Thanks!